app/HutchAgent/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "Serilog": {
-    "Using": ["Serilog.Sinks.Console"],
+    "Using": ["Serilog.Sinks.Console", "Serilog.Sinks.Seq"],
```

Comment:
I don't think Seq should be included by default as it requires the server. It should be opted into on environments that want it.

Related: This PR needs configuration documentation. Explain that Hutch *can* log to seq, and, if you want that, how to configure it.